packages/react-component-library/src/components/NumberInput/useValue.ts

```diff
@@ -25,6 +25,10 @@ export function useValue(value: number) {
     return nextValue || committedValue
   }, [committedValue, nextValue])
 
+  useEffect(() => {
+    setNextValue(value)
```

Comment:
Does it work if this is `setCommittedValue`? That's what worked for me locally

Reply:
That seems to have done the trick